components/ai_chat/content/browser/page_content_fetcher.h

```diff
@@ -17,9 +17,12 @@ class WebContents;
 namespace ai_chat {
 
 using FetchPageContentCallback =
-    base::OnceCallback<void(std::string, bool is_video)>;
+    base::OnceCallback<void(std::string page_content,
+                            bool is_video,
+                            std::string invalidation_token)>;
```

Comment:
why is the `invalidation_token` arg a string here and in other places string_view?

Reply:
They are different things. One is the string to use for comparison, owned by the caller and provided as `string_view`. And the other is the string provided for the next time it's called, meant to be given to the consumer so that it can outlive the function call, so it's provided as a `string`.

The `string_view` is an API function param and the `string` is a callback param.